src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/java/validators/annotations/AnnotationValidator.java

```diff
@@ -88,17 +89,16 @@ public void unregisterRule(AnnotationRule rule) {
 	 * @return the error message of the validation result of the attribute value and null otherwise.
 	 */
 	public String validate(String value, AnnotationAttributeRule rule) {
-		try {
-			if (rule == null) {
-				return null;
-			}
-			return rule.validate(value);
-		} catch (IndexNotReadyException | ProcessCanceledException | CancellationException e) {
-			throw e;
-		} catch (Exception e) {
-			LOGGER.log(Level.WARNING, e.getLocalizedMessage(), e);
-			return null;
-		}
+		return ExceptionUtil.executeWithExceptionHandling(
+				() -> {
+					if (rule == null) {
+						return null;
+					}
+					return rule.validate(value);
+				},
+				null,  // Fallback value in case of exception
```

Comment:
@mrglavas , Thanks for the suggestions, Yes you are correct that will get a `NullPointerException`.
So I initially have a thought of changing code like below

```
public String validate(String value, AnnotationAttributeRule rule) {
    return ExceptionUtil.executeWithExceptionHandling(
        () -> {
            if (rule == null) {
                return null;
            }
            return rule.validate(value);
        },
        () -> null,  // Fallback value supplier in case of exception
        e -> LOGGER.log(Level.WARNING, e.getLocalizedMessage(), e)
    );
}
```